apps/dashboard/src/components/auth/create-organization.tsx

```diff
@@ -122,9 +158,16 @@ export default function OrganizationCreate() {
         location: 'web',
         organizationId: organization.id,
         organizationName: organization.name,
+        region: selectedRegion,
       });
+
+      // Set the region metadata for the newly created organization
+      const regionMetadata = selectedRegion === 'singapore' ? 'ap-southeast-1' : 'us-east-1';
```

Comment:
What is happening with this regionMetadata? Seems like it's defined but not used. Also, any specific reason for the hardcoding? Can we make it more dynamic?>